index.js

```diff
@@ -69,6 +69,41 @@ class Executor {
         throw new Error('Not implemented');
     }
 
+    /**
+      * Starts a new periodic build in an executor
+      * @method _startPeriodic
+      * @param {Object} config               Configuration
+      * @param {Object} config.pipeline      Pipeline of the job
+      * @param {Object} config.job           Job object to create periodic builds for
+      * @param {Object} config.tokenGen      Function to generate JWT from username, scope and scmContext
+      * @param {Boolean}config.update        Boolean to determine if updating existing periodic build
+      * @return {Promise}
+      */
+    startPeriodic(config) {
+        return validate(config, executorSchema.start)
+            .then(validConfig => this._start(validConfig));
+    }
+
+    async _startPeriodic() {
+        throw new Error('Not implemented');
+    }
+
+    /**
+     * Stops a previously scheduled periodic build in an executor
+     * @async  _stopPeriodic
+     * @param  {Object}  config        Configuration
+     * @param  {Integer} config.jobId  ID of the job with periodic builds
+     * @return {Promise}
+     */
+    stopPeriodic(config) {
+        return validate(config, executorSchema.stop)
```

Comment:
Same issue here, it will check against https://github.com/screwdriver-cd/data-schema/blob/master/plugins/executor.js#L16-L19